www/includes/easyparliament/templates/html/search/form.php

```diff
@@ -12,7 +12,7 @@
                     <?php } ?>
                     <p>
                         <ul class="search-result-display-options">
-                        <li><a href="#options" class="search-options-toggle js-toggle-search-options">Advanced search</a></li>
+                        <li><a href="?show_advanced_options=true" class="search-options-toggle js-toggle-search-options">Advanced search</a></li>
```

Comment:
I am a bit worried that this will completely erase any advanced options the user has set elsewhere on the page.

(eg: if they perform an advanced search, with Javascript disabled, and then click the "Advanced search" link to show the form again and edit their parameters.) 

@dracos: Should we somehow concatenate `show_advanced_options=true` onto the end of whatever query parameters are already set?